prow/plugins/trigger/pr_test.go

```diff
@@ -147,3 +150,215 @@ func TestTrusted(t *testing.T) {
 		})
 	}
 }
+func TestHandlePullRequest(t *testing.T) {
+	var testcases = []struct {
+		name string
+
+		Author        string
+		ShouldBuild   bool
+		ShouldComment bool
+		HasOkToTest   bool
+		prLabel       string
+		prChanges     bool
+		prAction      github.PullRequestEventAction
+	}{
+		{
+			name: "Trusted user Open PR.",
```

Comment:
Ideally this says what is supposed to happen, not just the scenario.

AKA -- `Trusted user open PR should build`